modules/apps/content-dashboard/content-dashboard-journal-impl/src/test/java/com/liferay/content/dashboard/journal/internal/item/JournalArticleContentDashboardItemTest.java

```diff
@@ -19,17 +19,25 @@
 import com.liferay.content.dashboard.item.ContentDashboardItemVersion;
```

Comment:
How do you feel about moving all the unit tests to integration tests?

I think as we don't need to use mocks maybe we could read them more easily?

Reply:
Me too :).